homeassistant/components/websocket_api/commands.py

```diff
@@ -413,7 +451,7 @@ async def handle_subscribe_trigger(hass, connection, msg):
     trigger_config = await trigger.async_validate_trigger_config(hass, msg["trigger"])
 
     @callback
-    def forward_triggers(variables, context=None):
+    def forward_triggers(variables: Any, context: Any = None) -> None:
```

Comment:
I wasn't able to figure out what's passed here. This callback is passed as `action` to `async_attach_trigger` in trigger platform.

Reply:
I think the `variables` is a `dict` and the context is a core `Context`.

Here's an example of a call:
https://github.com/home-assistant/core/blob/73d7a754e846eff872ccb9b82284d317de2c71bd/homeassistant/components/homeassistant/triggers/state.py#L133-L148

Reply:
Added, thanks!